src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/AuthenticationHelper.NtAuth.cs

```diff
@@ -77,7 +78,23 @@ private static async Task<HttpResponseMessage> SendWithNtAuthAsync(HttpRequestMe
 
                         string challengeData = challenge.ChallengeData;
 
-                        string spn = "HTTP/" + authUri.IdnHost;
+                        // Need to use FQDN normalized host so that CNAME's are traversed.
```

Comment:
Should we move the comment rather into the else-branch?

Reply:
I think the comment is in the right place since it describes the logic to calculate the SPN.  But I will revise the comment because it needs to describe that we skip DNS lookup in cases of IP literals.